src/combinatorics/set_family/compression/uv.lean

```diff
@@ -187,6 +200,43 @@ begin
   { exact (ha.2 ha.1).elim }
 end
 
+lemma le_of_mem_compression (h : a ∈ 𝓒 u v s) (ha : a ∉ s) : u ≤ a :=
+begin
+  rw mem_compression at h,
+  obtain _ | ⟨-, b, hb, hba⟩ := h,
+  { cases ha h.1 },
+  unfold compress at hba,
+  split_ifs at hba,
+  { rw [←hba, le_sdiff],
+    exact ⟨le_sup_right, h.1.mono_right h.2⟩ },
+  { cases ne_of_mem_of_not_mem hb ha hba }
+end
+
+lemma disjoint_of_mem_compression (h : a ∈ 𝓒 u v s) (ha : a ∉ s) : disjoint v a :=
```

Comment:
This could also be `disjoint_of_not_subset`, right?

Reply:
Do you mean `_of_not_mem`? At any rate,  I changed the lemma name to match the other ones around.

Reply:
Sorry, my comment was useless; I didn't notice that `a` was used in the conclusion too.